client/modules/IDE/components/NewFolderForm.jsx

```diff
@@ -32,6 +33,7 @@ class NewFolderForm extends React.Component {
           {...domOnlyProps(name)}
         />
         <input type="submit" value="Add Folder" aria-label="add folder" />
```

Comment:
Maybe it would also make sense to disable the "submit" button if the form hasn't been touched, see https://github.com/processing/p5.js-web-editor/blob/master/client/modules/User/components/LoginForm.jsx#L33